templates/module.j2

```diff
@@ -1,5 +1,5 @@
 {% extends "base.j2" %}
-{% block title %}{{ filename | filename_import | htmlify_name }}{% endblock %}
+{% block title %}{{ filename | filename_import | e }}{% endblock %}
```

Comment:
Unfortunately we also need linebreaks in module names.  See e.g. this overly long name here: https://leanprover-community.github.io/mathlib_docs/category_theory/limits/shapes/constructions/limits_of_products_and_equalizers.html

I would suggest to have `htmlify_name` return a `<span class="name">....</span>` and replace `.decl_name > a` by `.name` in the CSS.

Reply:
Ah thanks @gebner - I hadn't thought about breaks in module names 🙂 . I just pushed a fix